README.md

```diff
@@ -3,7 +3,7 @@
 - Install Node.js & NPM (https://nodejs.org/en/).
 - Install Yarn (if you prefer this instead of NPM) (https://yarnpkg.com/en/).
 
-## Usage
+## Usage Without Docker
```

Comment:
Change `Without` with `without`. It's a conjunction in English, so we don't need to use uppercase word here.